rllib/offline/estimators/fqe_torch_model.py

```diff
@@ -113,14 +115,18 @@ def train_q(self, batch: SampleBatch) -> TensorType:
             A list of losses for each training iteration
         """
         losses = []
+        if self.minibatch_size is None:
+            minibatch_size = batch.count
         for _ in range(self.n_iters):
             minibatch_losses = []
             batch.shuffle()
-            for idx in range(0, batch.count, self.batch_size):
-                minibatch = batch[idx : idx + self.batch_size]
+            for idx in range(0, batch.count, minibatch_size):
+                minibatch = batch[idx : idx + minibatch_size]
                 obs = torch.tensor(minibatch[SampleBatch.OBS], device=self.device)
```

Comment:
**todo**: We need to write GPU tests for OPE methods to make sure we don't get device mismatch / OOM errors. We'll do it in the follow up PRs.

Reply:
I added `RLLIB_NUM_GPUS` to the `.resources` for test_ope.py. We can probably run the same tests twice; once on cpu and once with `RLLIB_NUM_GPUS=1`.

Reply:
yes, but the build info should also be updated with a "gpu" tag to spawn a gpu machine for the CI.

Reply:
Again, let's focus on this on the next PR.